trpcultivate_phenotypes/src/Plugin/Validators/ValidDelimitedFile.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/**
+ * @file
+ * Contains validator plugin definition.
+ */
+
+namespace Drupal\trpcultivate_phenotypes\Plugin\Validators;
+
+use Drupal\trpcultivate_phenotypes\TripalCultivateValidator\TripalCultivatePhenotypesValidatorBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Validate a line in a data file is properly delimited.
+ *
+ * @TripalCultivatePhenotypesValidator(
+ *   id = "trpcultivate_phenotypes_validator_valid_delimited_file",
```

Comment:
I think we are moving away from these long ids for validators since many of them will be used across multiple modules in the future. Here is my suggestion to keep consistent with the validators I've been working on:
```suggestion
 *   id = "valid_delimited_file",
```